src/open_inwoner/accounts/tests/test_oidc_views.py

```diff
@@ -643,20 +645,64 @@ def test_logout(self, mock_get_solo):
         self.assertFalse(User.objects.filter(email="new_user@example.com").exists())
 
         # enter the logout flow
-        with requests_mock.Mocker() as m:
-            m.post("http://localhost:8080/logout")
-            logout_response = self.client.get(logout_url)
+        logout_response = self.client.get(logout_url)
 
-            self.assertEqual(len(m.request_history), 1)
-            self.assertEqual(m.request_history[0].url, "http://localhost:8080/logout")
-            self.assertEqual(m.request_history[0].body, "id_token_hint=foo")
+        self.assertRedirects(
+            logout_response,
+            "http://localhost:8080/logout"
+            + "?"
+            + urlencode(
+                dict(
+                    id_token_hint="foo",
+                    post_logout_redirect_uri=f"http://testserver{settings.LOGOUT_REDIRECT_URL}",
+                )
+            ),
+            fetch_redirect_response=False,
+        )
+
+        self.assertNotIn("oidc_states", self.client.session)
+        self.assertNotIn("oidc_id_token", self.client.session)
+        self.assertFalse(logout_response.wsgi_request.user.is_authenticated)
```

Comment:
Here and below: why assert about `wsgi_request.user` as opposed to `user` (defined earlier in the function?)